src/foundry/foundry.js/pixi/points/wallEndpoint.d.ts

```diff
@@ -1,88 +1,92 @@
-/**
- * A special subclass of PIXI.Point which is used for modeling Wall endpoints.
- * A wall endpoint must have integer coordinates.
- *
- * This was used for the RadialSweepPolygon but can now be deleted once that is
- * @deprecated since v9d2
- */
-declare class WallEndpoint extends PIXI.Point {
-  /**
-   * @param x - The integer x-coordinate
-   * @param y - The integer y-coordinate
-   */
-  constructor(x: number, y: number);
+import type { ConfiguredObjectClassOrDefault } from '../../config.js';
```

Comment:
Please use either `ConfiguredObjectClassForName` or `ObjectClass` from the `helperTypes.d.ts` instead.